pyproject.toml

```diff
@@ -17,9 +20,11 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.ruff]
 select = ["ALL"]
+ignore = ["ANN101", "ANN102"]  # Type annotations for 'self' and 'cls'
 fixable = ["ALL"]
-fix = true
+fix = false
```

Comment:
Thought we were defaults all on kinda people?!

Reply:
I changed my mind regarding auto-fixing on save due to the remove-unused-imports feature. I have the habit of hitting CTRL + S quite frequently. Therefore, when I import a new module and (due to my habit) save the change, it will remove my import before I use it. Instead, I often press CTRL + SHIFT + P to run Ruff in auto-fix mode. See:
![image](https://github.com/leoschleier/pyper/assets/43878374/8dea8827-b6ec-4029-a416-d877b9e31213)

Reply:
The golang extension has the same behaviour and initially I shared your opinion. However, after I got used to it, I loved this feature!